src/core/renderController.ts

```diff
@@ -89,23 +84,24 @@ module Plottable {
     export function flush() {
       if (_animationRequested) {
         // Layout
-        _componentsNeedingComputeLayout.values().forEach((component: Component) => component.computeLayout());
+        _componentsNeedingComputeLayout.values().forEach((component) => component.computeLayout());
```

Comment:
From offline:
Going forward, we'll avoid typing things when the tools can correctly infer the types.